docs/settingup.md

```diff
@@ -687,7 +687,7 @@ Publish a bookmark.
 
 ### Example
 
-Publish bookmark previously created in the script with an id `bookmark1`.
+Publish the bookmark with id "bookmark1" that was created earlier on in the script.
```

Comment:
Could you please "highlight" id (`id`), so that it corresponds to the way it is presented under Settings?